source/shared/cpp/ObjectModel/Container.h

```diff
@@ -24,24 +24,16 @@ namespace AdaptiveSharedNamespace
         std::vector<std::shared_ptr<BaseCardElement>>& GetItems();
         const std::vector<std::shared_ptr<BaseCardElement>>& GetItems() const;
 
-        ContainerStyle GetStyle() const;
-        void SetStyle(const ContainerStyle value);
-
         std::shared_ptr<BaseActionElement> GetSelectAction() const;
         void SetSelectAction(const std::shared_ptr<BaseActionElement> action);
 
         void SetLanguage(const std::string& value);
```

Comment:
Can language and select action go up to the base class as well?

Reply:
I also think that the suggestion makes code cleaner and organized.

I didn't included them initially because I felt  weird to include the properties  that are not essential to collection type. FactSet and TextBlock, for example, have language properties.



---
In reply to: [258664311](https://github.com/Microsoft/AdaptiveCards/pull/2406#discussion_r258664311) [](ancestors = 258664311)